README.md

```diff
@@ -155,8 +156,13 @@ AWS Media Services Application Mapper consists of:
 <a name="license"></a>
 # License
 
-See license [here](https://github.com/awslabs/aws-media-services-application-mapper/blob/main/LICENSE.txt).
+See license [here](LICENSE.txt).
 
 
+<a name="metrics"></a>
+# Anonymous Metrics 
```

Comment:
We should probably also link to the information right on this README on how to enable/disable this since the IG might not be available with updated info as soon as we release.

Reply:
I'll add this to README.md.